src/io/terminal.cpp

```diff
@@ -57,6 +57,11 @@ void terminal_set_color(uint8_t color) {
 
 void terminal_put_entry_at(char c, uint8_t color, size_t x, size_t y) {
 	const size_t index = terminal_buffer_idx(x,y);
+	if(c = 0xa){
```

Comment:
Are you comparing this to see whether c is a newline? If so, it would be easier for other people to read if you compare c against the actual character and not the value.

Also because you have a single equals, in C this will result in this statement assigning 0xa to c, which will cause the resulting if statement to always evaluate to true.

As a style nitpick, I would say that most times there are more spaces in-between the pieces of the if statement.

All in all, my recommendation `if (c == '\n') {`

Reply:
Fixed in commit e1a4e6800932643864f58ac7540cfb09a4cdbb2c. 

https://github.com/Voluptuous-Rice-Box/KROS/pull/13/commits/e1a4e6800932643864f58ac7540cfb09a4cdbb2c

Surprised my terminal didn't end up filled with just newlines.